jwst/pipeline/calwebb_spec2.py

```diff
@@ -491,7 +495,6 @@ def _process_grism(self, data):
         calibrated = self.barshadow(calibrated)
         calibrated = self.wfss_contam(calibrated)
         calibrated = self.photom(calibrated)
-        calibrated = self.pixel_replace(calibrated)
```

Comment:
Ah, ignore my previous question. I see that `pixel_replace` used to be called from these lower-level functions, but has now intentionally been moved up into the main pipeline flow.